### PR TITLE
Replace theme conditionals in templates with new skyIfTheme directive

### PR DESCRIPTION
**Pull request by johnhwhite**
- Replace theme conditionals in templates with new skyIfTheme directive

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-replace-conditionals-with-skyiftheme)

File: src/app/docs/modal/modal-docs.component.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
         >
         </sky-docs-demo-control-panel-radio-group>
       </sky-docs-demo-control-panel-section>
-      <sky-docs-demo-control-panel-section *ngIf="(themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'default'">
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/modal/modal-docs.component.html:23]: You can remove `themeSvc` from the `.ts` file.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**